front/app/components/SignUpIn/SignUpInModal.tsx

```diff
@@ -81,6 +83,9 @@ const SignUpInModal = memo<Props>(({ className, onMounted }) => {
       signUpActiveStepChange$.subscribe(({ eventValue: activeStep }) => {
         setSignUpActiveStep(activeStep);
       }),
+      changeMetaData$.subscribe(({ eventValue: metaData }) => {
+        setMetaData(metaData);
+      }),
```

Comment:
(not blocking, but rather a suggestion) It's somewhat of a structural change I propose, but perhaps it makes sense to have a unified `setMetaData` function and `metaData$` observable in events.ts that would be the single source of truth for the signUpIn metadata object, and which each all components in the SignUpIn folder would call/subscribe to initialize, alter, reset or read the metaData. Now I have the feeling it's a bit scattered and we have several ways of interacting with the metaData, which might be a bit confusing and not easily scalable.

Reply:
You can take a look at `containers\ProjectsShowPage\timeline\events.ts` to see an example of what I mean. In that events.ts file we have `selectPhase` function and `selectedPhase$` observable which is the single source of truth for the selected timeline phase on the project page.

Reply:
Yeah sure, definitly ok. Mentioned it more as a possible future improvement than something that would have to change now